### ISSUE TITLE
Git Repo: Typo in TAG name "v2.7,0" should be "v2.7.0"  AND  "origin/r2.6.1" likely branched from wrong commit, missing r2.6.0 changes

### ISSUE THREAD

**Issue by scottc21** [keras-team/keras#15583]
**System information**.
N/A -- this is a GIT issue with Labels and branches ( see below )

The description below and attached screenshots should make this pretty clear what I am talking about.

**Describe the problem**.
Item #1  -- Typo in TAG name "v2.7,0" should be "v2.7.0"
Based on previous TAG names, it appears that ""v2.7,0" is a typo  ( comma vs period )

Item #2  -- "origin/r2.6.1" likely branched from wrong commit, missing r2.6.0 changes
If you view the tree view in Git Client, you will noticed that the "origin/r2.6.1" was likely branched from the wrong changelist.  There are 5 items in the "r2.6.0" branch that will be lost from "origin/r2.6.1" and this appears to be an accident.

**Describe the current behavior**.
N/A -- this is a GIT issue and not a Keras bug

**Describe the expected behavior**.
Item #1  -- Typo in TAG name "v2.7,0" should follow the previous pattern and be updated to be "v2.7.0"

Item #2 -- Review and likely re-branch  "origin/r2.6.1" from the correct change-list --  I assume branched from  the TAG "v2.6.0"

**Standalone code to reproduce the issue**.
N/A -- Git issue, not a Keras bug

![Keras_branch_2 6 1_Likely_Branched_at_wrong_location](https://user-images.githubusercontent.com/93684290/140200333-bc3f9e49-7207-4ccb-8a26-15913e91d5c7.png)
**Source code / logs**.
See GIT tree view in Git Client -- this is pretty obvious to see what I am describing. 
![Keras_TAG_typo__2 7,0__view_01_](https://user-images.githubusercontent.com/93684290/140200347-adca66bd-b6eb-403a-8569-1d4a8789c7be.png)
![Keras_TAG_typo__2 7,0__view_02_Not_following_new_pattern_](https://user-images.githubusercontent.com/93684290/140200355-9176103b-7526-4739-87ad-8c06f5e1f23c.png)

**Comment by scottc21** [comment]
On Item 2 ( 2.6 series):   it appears the TensorFlow team is getting ready to release TensorFlow 2.6.2 based on the changesets from today (  11/03/2021 01:34 PM )  --  Update version numbers for TensorFlow 2.6.2.

Also, there is another 'keras' bug report ( #15579 ) that discusses some of the 2.6.x 2.7.x  release issues.  See link below
-- https://github.com/keras-team/keras/issues/15579

**Comment by qlzh727** [comment]
Thanks for the issue. 

The github tag has been fixed last night, see https://github.com/keras-team/keras/tree/v2.7.0.

The v2.6.1 was just for testing and not used for any release. We can actually remove it. The Keras 2.6 was released based on r2.6.0 branch.
